### PR TITLE
Ret 3740 - add Welsh translations

### PR DESCRIPTION
**Pull request by lvo700**
### JIRA link

https://tools.hmcts.net/jira/browse/RET-3740

### Change description

Add Welsh translations for Rule 92 page.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/resources/locales/cy/translation/common.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "continue": "Parhau",
   "view": "Gweld",
   "change": "Newid",
-  "cancel": "Welsh translation required",
+  "cancel": "Allgofnodi",
```
Comment (marcjudge) [inline src/main/resources/locales/cy/translation/common.json:7]: Don't think this is correct. This means 'Sign out'

Comment (lvo700) [reply src/main/resources/locales/cy/translation/common.json]: Fixed

**Review by brendanmchugh — APPROVED** [review]

File: src/main/resources/locales/cy/translation/contact-the-tribunal.json [hunk 2]
```diff
@@ -1,104 +1,104 @@
 {
-  "title": "Welsh Translation required",
-  "h1": "Welsh Translation required",
+  "title": "Cysylltu â’r tribiwnlys ynglŷn â’ch achos",
```
Comment (marcjudge) [inline src/main/resources/locales/cy/translation/contact-the-tribunal.json:2]: As you added translations for the contact the tribunal page, could you please add the feature flag to the controller?

**Review by marcjudge — CHANGES_REQUESTED** [review]

File: src/main/resources/locales/cy/translation/contact-the-tribunal.json (hunk 2 above)
Comment (lvo700) [reply src/main/resources/locales/cy/translation/contact-the-tribunal.json:2]: Added feature flag

**Review by marcjudge — APPROVED** [review]

**Review by brendanmchugh — APPROVED** [review]

**Pull request merged**